src/Raven.Studio/typescript/models/resources/info/databaseInfo.ts

```diff
@@ -44,6 +44,34 @@ class databaseInfo extends resourceInfo {
         this.documentsCount(databaseInfo.DocumentsCount);
         this.indexesCount(databaseInfo.IndexesCount);
     }
+
+    static empty(resourceName: string): databaseInfo {
```

Comment:
It shouldn't:
In general we should call /resources?name={dbName}&type=db
but this call can't wake database up ( and I have hope it is not). 

In v3.5 we fetched information using /stats endpoint, and it was waken up database. In v4.0 we have different behavior.